test/Microsoft.ML.Tests/Scenarios/Api/IntrospectiveTraining.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.FastTree;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Internal.Internallearn;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.Runtime.TextAnalytics;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.ML.Tests.Scenarios.Api
+{
+
+    public partial class ApiScenariosTests
+    {
+        private TOut GetValue<TOut>(Dictionary<string, object> keyValues, string key)
+        {
+            if (keyValues.ContainsKey(key))
+                return (TOut)keyValues[key];
+
+            return default(TOut);
+        }
+
+        /// <summary>
+        /// Introspective training: Models that produce outputs and are otherwise black boxes are of limited use;
+        /// it is also necessary often to understand at least to some degree what was learnt. To outline critical
+        /// scenarios that have come up multiple times:
+        ///  *) When I train a linear model, I should be able to inspect coefficients.
+        ///  *) The tree ensemble learners, I should be able to inspect the trees.
+        ///  *) The LDA transform, I should be able to inspect the topics.
+        ///  I view it as essential from a usability perspective that this be discoverable to someone without 
+        ///  having to read documentation.E.g.: if I have var lda = new LdaTransform().Fit(data)(I don't insist on that
```

Comment:
>Fit [](start = 87, length = 3)

Do we already have Fit() implemented?

Reply:
This is old Api examples, Fit is part of new Api